lib/view_component/base.rb

```diff
@@ -156,6 +156,25 @@ def request
     mattr_accessor :test_controller
     @@test_controller = "ApplicationController"
 
+    module CompileCache
```

Comment:
`base.rb` is getting pretty big! What do you think about putting this in a separate file and including it?